modules/doi/includes/utilities.inc

```diff
@@ -202,39 +202,36 @@ function islandora_doi_crossref_translator($crossref_data) {
       foreach ($metadata->childNodes as $child) {
         switch ($child->nodeName) {
           case 'full_title':
-            $titleinfo = $mods->createElement('titleInfo');
-            $relateditem->appendChild($titleinfo);
-            $title = $mods->createElement('title');
-            $titleinfo->appendChild($title);
             if (!is_null($child->firstChild)) {
-              $title_text = $mods->importNode($child->firstChild);
-              $title->appendChild($title_text);
+              $titleinfo = $mods->createElement('titleInfo');
+              $relateditem->appendChild($titleinfo);
+              $title_text = islandora_doi_filter_html($crossref_xml->saveXML($child->firstChild));
```

Comment:
If $child (aka the 'full_title' node) contains markup, then it contains multiple children, and passing `$crossref_xml->saveXML($child->firstChild)` to the filter function will lose anything after the first child node (probably a text node).

Reply:
Forgive my ignorance, @rosiel, but would this not have prevented the markup to show up in the old code, as well?

Reply:
@d-r-p jinx. I was writing a little test to make a side by side comparison. Just to be clear, first child, if dumped as XML is <full_title>bla bl<i>curvy blabbla</i></full_title> right?

Reply:
Ok, so I did side by side code test (please run locally, pretty simple) and now i don't see how the original code is introducing HTML based children. See https://github.com/Islandora/islandora_scholar/pull/288#issuecomment-392833469